ui/src/main/js/page/channel/azure/AzureBoardsPageForm.js

```diff
@@ -174,7 +175,7 @@ const AzureBoardsForm = ({ csrfToken, errorHandler, readonly, displayTest }) =>
                     id={AZURE_BOARDS_GLOBAL_FIELD_KEYS_UPDATED.configureOAuth}
                     name={AZURE_BOARDS_GLOBAL_FIELD_KEYS_UPDATED.configureOAuth}
                     label="Microsoft OAuth"
-                    buttonLabel="Authenticate & Save"
+                    buttonLabel="Save Authenticate"
```

Comment:
Should it be 'Save Authentication' or 'Save/Authenticate' The text 'Save Authenticate' doesn't seem right to me.

Reply:
I'm with Paulo, 'Save & Authenticate' or 'Save/Authenticate' would be better. Functionally under the hood you aren't saving the authentication. You perform a save action, then an authenticate action.

Reply:
Great catch.  Just updated the text to 'Save/Authenticate'